src/rpc/core_rpc_server.h

```diff
@@ -129,6 +129,7 @@ namespace cryptonote
         MAP_JON_RPC_WE("getblocktemplate",       on_getblocktemplate,           COMMAND_RPC_GETBLOCKTEMPLATE)
         MAP_JON_RPC_WE("submit_block",           on_submitblock,                COMMAND_RPC_SUBMITBLOCK)
         MAP_JON_RPC_WE("submitblock",            on_submitblock,                COMMAND_RPC_SUBMITBLOCK)
+        MAP_JON_RPC_WE("generateblocks",         on_generateblocks,             COMMAND_RPC_GENERATEBLOCKS)
```

Comment:
Maybe make this a restricted command. Doesn't matter a whole lot though since it errors out when not in fake mode.

Reply:
This was a new element for me! Doesn't seem to hurt to add it